pom.xml

```diff
@@ -446,12 +440,6 @@
             <version>${netty.version}</version>
             <scope>compile</scope>
         </dependency>
-        <dependency>
```

Comment:
We're not getting rid of `junit-jupiter-params`, the reason it's removed is because it was duplicated.